specification/metrics/semantic_conventions/runtime-environment-metrics.md

```diff
@@ -36,6 +36,10 @@ semantic conventions when instrumenting runtime environments.
   * [Metric: `process.runtime.jvm.buffer.usage`](#metric-processruntimejvmbufferusage)
   * [Metric: `process.runtime.jvm.buffer.limit`](#metric-processruntimejvmbufferlimit)
   * [Metric: `process.runtime.jvm.buffer.count`](#metric-processruntimejvmbuffercount)
+  * [Metric: `process.runtime.jvm.cpu.monitor.duration`](#metric-processruntimejvmcpumonitorduration)
```

Comment:
You'll want to rebase and we should decide whether any of these should be included in the effort for stable semantic conventions, or be experimental for now and followup stabilizing them later.

Reply:
I brought it up here https://github.com/open-telemetry/opentelemetry-specification/issues/3419 but it seemed like the plan was to exclude them from initial stability. I think these metrics would be quite useful to have, but I guess are not worth delaying stabilization for since they only apply to Java 14+.